static/sidebar/js/modal/index.js

```diff
@@ -54,14 +54,19 @@
       prodNode.href = productLink;
       prodNode.innerHTML = '<div class="tvp-product-image" '+productImgStyle+'><div/>';
       thumbsFrag.appendChild(prodNode);
+      
+      var trimmedTitle = product.title.length > 50 ? product.title.substring(0, 50 - 3) + "..." :product.title;
```

Comment:
handle the qty like, custom from the settings?
the "-3" was for the dots, but i guess is not necessary.